go/install/install.go

```diff
@@ -91,6 +91,8 @@ func (c ComponentName) Description() string {
 		return "Fuse"
 	case ComponentNameHelper:
 		return "Privileged Helper Tool"
+	case ComponentNameKBNM:
+		return "Chrome Native Messaging"
```

Comment:
FWIW it's not just for Chrome, but for now it is. Maybe "Browser Native Messaging" is better.